csfieldguide/chapters/content/en/human-computer-interaction/sections/user-experience.md

```diff
@@ -0,0 +1,559 @@
+# User Experience
+
+So far in this chapter we have gone into detail about HCI and the principles behind it.
+In the tech industry it is not very often that you meet someone with a job title of “HCI expert”, instead, they are called *User Experience (UX) Designers*, and HCI is recognised as their academic background.
+UX Designers look at the big picture, they think about everything from a user’s first encounter, to their day to day interactions with it, how it affects their job or life, and even how they feel about it.
+In contrast, HCI tends to focus on specific tasks and elements of the interface.
+
+User Experience is all about *how* the user interacts with the device or application.
```

Comment:
Good place for glossary definition of 'user experience'

Reply:
This is the chapter about UX, do we need to make a separate glossary entry?

Reply:
I think it's worthwhile. Ideally all keys words are defined in our glossary.